src/others/file_explorer/lv_file_explorer.c

```diff
@@ -512,8 +531,8 @@ static void browser_file_event_handler(lv_event_t * e)
             show_dir(obj, (char *)file_name);
         }
         else {
```

Comment:
Here we can use the second column of the table to determine whether or not the selected item is a directory.